src/index.ts

```diff
@@ -322,7 +322,7 @@ export class SentryPlugin implements Plugin {
   async _resolveGitRefs(gitRev: GitRev, release: SentryRelease): Promise<SentryRelease> {
     const origin = await gitRev.origin();
     const commit = await gitRev.long();
-    let repository = /[:/]([^/]+\/[^/]+?)(?:\.git)?$/i.exec(origin)?.[1];
+    let repository = /[:/](.+?)(?:\.git)?$/i.exec(origin)?.[1];
     if (repository && origin.includes("gitlab")) {
```

Comment:
Did you try if this block is still necessary? I would assume we can remove the code altogether.

Reply:
no, not really, I have just quickly tested that regular expression works for my case...